noirup/noirup

```diff
@@ -3,6 +3,7 @@ set -e
 
 NARGO_DIR=${NARGO_DIR-"$HOME/.nargo"}
 NARGO_BIN_DIR="$NARGO_DIR/bin"
+CONFIG_DIR=${XDG_CONFIG_HOME-"$HOME/.config"}
```

Comment:
This location is only valid for Linux, on darwin it will failr as it expects `$HOME/Library/Application Support`